server/src/routes/routes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express'
+
+const router = express.Router()
+
+router.use('/version', require('./version'))
+router.use('/users', require('./users'))
+router.use('/', require('./comments'))
```

Comment:
Why are some routes mounted on root and some are routed under a path?

Reply:
Comments.js contains endpoints like questions, comments, reactions. Should we separate them to different files and make files for each one of them.

Reply:
done